docs/trace/getting-started/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Diagnostics;

```

Comment:
I was thinking that we might need to have a CI job to ensure whatever code snippet we have in the Markdown files are actually up-to-date.

Reply:
I think docs.microsoft do something a little different. The snippets aren't hard-coded in markdown files, they are in project and somehow they make their reference to the project/file/section.

Reply:
This can be done while it would introduce other maintenance issue. We can figure out which way is better.

Reply:
I tinkered a little bit with Try.NET awhile ago. Seems pretty slick and integrates easily with CI making certain that markdown files that reference the snippets remain in sync. Anyone have more experience with this?